Error on watch process

I receive the following error every time gulp watch runs.

```
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: watch ENOENT
    at errnoException (fs.js:1030:11)
    at FSWatcher.start (fs.js:1062:11)
    at Object.fs.watch (fs.js:1087:11)
    at Gaze._watchDir (/opt/app/node_modules/gulp/node_modules/vinyl-fs/node_modules/glob-watcher/node_modules/gaze/lib/gaze.js:289:30)
    at /opt/app/node_modules/gulp/node_modules/vinyl-fs/node_modules/glob-watcher/node_modules/gaze/lib/gaze.js:358:10
    at iterate (/opt/app/node_modules/gulp/node_modules/vinyl-fs/node_modules/glob-watcher/node_modules/gaze/lib/helper.js:52:5)
    at /opt/app/node_modules/gulp/node_modules/vinyl-fs/node_modules/glob-watcher/node_modules/gaze/lib/helper.js:61:11
    at /opt/app/node_modules/gulp/node_modules/vinyl-fs/node_modules/glob-watcher/node_modules/gaze/lib/gaze.js:420:5
    at iterate (/opt/app/node_modules/gulp/node_modules/vinyl-fs/node_modules/glob-watcher/node_modules/gaze/lib/helper.js:52:5)
    at /opt/app/node_modules/gulp/node_modules/vinyl-fs/node_modules/glob-watcher/node_modules/gaze/lib/helper.js:61:11

npm ERR! Linux 3.13.0-35-generic
npm ERR! argv "node" "/usr/local/bin/npm" "run" "watch"
npm ERR! node v0.10.32
npm ERR! npm  v2.1.14
npm ERR! code ELIFECYCLE
npm ERR! nyc-trees@0.0.0 watch: `sudo -u nyc-trees ./node_modules/.bin/gulp watch --debug`
npm ERR! Exit status 8
npm ERR!
npm ERR! Failed at the nyc-trees@0.0.0 watch script 'sudo -u nyc-trees ./node_modules/.bin/gulp watch --debug'.
npm ERR! This is most likely a problem with the nyc-trees package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     sudo -u nyc-trees ./node_modules/.bin/gulp watch --debug
npm ERR! You can get their info via:
npm ERR!     npm owner ls nyc-trees
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /opt/app/npm-debug.log
Connection to 127.0.0.1 closed.
```

![image](https://cloud.githubusercontent.com/assets/1809908/7433848/77ed40f8-f003-11e4-87ec-10aec4f6da54.png)

--- a/src/nyc_trees/gulpfile.js
+++ b/src/nyc_trees/gulpfile.js
@@ -238,9 +238,9 @@ gulp.task('watch', ['watchify'], function() {
     // Note: JS rebuilding is handled by watchify, in order to utilize it's
     // caching behaviour
     livereload.listen({auto: true });
-    gulp.watch('sass/**/*.scss', ['sass']);
-    // Rerun collectstatic whenever files are added to the static files dir
-    gulp.watch(intermediaryDir + '**', ['collect-debug']);
+    gulp.watch('sass/**/*.scss', function(cb) {
+        runSequence('sass', 'collect-debug');
+    });
 });
 
 gulp.task('watch-tests', function() {

